.env.example

```diff
@@ -1,12 +1,15 @@
-API_URL=http://localhost:8000/api/v1
-S3_URL=https://s3.amazonaws.com
+# Assume that we aren't working on the navigator-backend project
+# If you are swap this out for
+# API_URL=http://localhost:8888/api/v1
+API_URL=https://app.dev.climatepolicyradar.org/api/v1
+S3_URL=https://cpr-staging-targets-json-store.s3.eu-west-1.amazonaws.com
 
 THEME=cpr
 
-ADOBE_API_KEY=1234567890
+ADOBE_API_KEY=dca9187b65294374a6367824df902fdf
 
 REDIRECT_FILE="test.json"
 
-HOSTNAME=https://localhost:3000
+HOSTNAME=http://localhost:3000
 
-NEXT_PUBLIC_APP_TOKEN=some_token
+NEXT_PUBLIC_APP_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhbGxvd2VkX2NvcnBvcmFfaWRzIjpbIkNDTFcuY29ycHVzLmkwMDAwMDAwMS5uMDAwMCIsIkNQUi5jb3JwdXMuaTAwMDAwMDAxLm4wMDAwIiwiVU5GQ0NDLmNvcnB1cy5pMDAwMDAwMDEubjAwMDAiXSwiZXhwIjoyMDQyMTEzMzY5LCJpYXQiOjE3MjY1NzY5NjksImlzcyI6IkNsaW1hdGUgUG9saWN5IFJhZGFyIiwic3ViIjoiQ1BSIiwiYXVkIjoiaHR0cHM6Ly9hcHAuZGV2LmNsaW1hdGVwb2xpY3lyYWRhci5vcmcvIn0.mJ2qLJmMyPLGt0rM_tTXhlVv1glxooxmQV0bWrvPwKU
```

Comment:
[These are already exposed here](https://github.com/climatepolicyradar/navigator-frontend/blob/main/Makefile#L3-L8).

Reply:
Ok, didn't know that. Just wanted to make sure we're all aware and agreed on this.